tests/classdefs.pyi

```diff
@@ -34,7 +37,7 @@ class Good:
     def __ior__(self: Self, other: Self) -> Self: ...
 
 class Fine:
-    @overload
+    @typing_extensions.overload
```

Comment:
Mixing `typing.overload` and `typing_extensions.overload` in the same set of overloads isn't a great idea. Could you put them on separate functions instead?

We could also lint against this, but (a) it feels unlikely to happen in practice so may not be worth spending time on and (b) it can happen outside of stubs too so flake8-pyi wouldn't be the best place for a check.

Reply:
> Mixing `typing.overload` and `typing_extensions.overload` in the same set of overloads isn't a great idea. Could you put them on separate functions instead?

Yeah, good point, I'll make the change!